src/dataframe/dataframe.jl

```diff
@@ -499,6 +499,12 @@ Base.setproperty!(df::DataFrame, col_ind::Symbol, v::AbstractVector) =
     (df[!, col_ind] = v)
 Base.setproperty!(df::DataFrame, col_ind::AbstractString, v::AbstractVector) =
     (df[!, col_ind] = v)
+Base.setproperty!(::DataFrame, col_ind::Symbol, v::Any) =
```

Comment:
`col_ind` and `v` aren't used.

Reply:
I added them because they are referenced in the error message, so I thought to add them, as when the user later goes to the method definition it is clear what refers to what (but if you prefer I can remove them).

Reply:
I don't really mind.